spec/requests/api/create_article_spec.rb

```diff
@@ -0,0 +1,50 @@
+RSpec.describe 'POST /api/articles', type: :request do
+  subject { response }
+  describe 'when the article is successfully created' do
+    before do
+      post '/api/articles',
+           params: { title: 'Amazing title',
+                     lede: 'Amazing lede...' }
+    end
+
+    it { is_expected.to have_http_status 201 }
+
+    it 'is expected to return a response message when an article is created' do
```

Comment:
describe and it blocks should form one sentence. In this case it will be better to have `it is expected to respond with success message`

Reply:
Spec has been modified accordingly.